common/app/views/support/AmpAd.scala

```diff
@@ -4,7 +4,7 @@ import model.Tag
 import model.Article
 import play.api.libs.json.{JsObject, Json}
 
-object AmpAd {
+case class AmpAd(article: Article, uri: String, edition: String) {
```

Comment:
I guess you can also add the `toString` in this class, so you don't need to explecitely do `toJson().toString` every time you need a string